packages/curve-ui-kit/src/widgets/Disclaimer/HeaderPage.tsx

```diff
@@ -15,12 +15,9 @@ const { Spacing, Sizing, MaxWidth } = SizesAndSpaces
 export const HeaderPage = () => {
   const router = useRouter()
 
-  const [canGoBack, setCanGoBack] = useState(false)
-  // Check if window.history has previous entries from our app
-  // TODO: this doesn't work properly yet.
-  useEffect(() => {
-    setCanGoBack(window.history.length > 1 && document.referrer.includes(window.location.host))
-  }, [])
+  // Check if window.history has previous entries from our app.
+  // TODO: this doesn't work properly yet, but also isn't used yet.
```

Comment:
why isn't this used? Can we delete it?

Reply:
It's this 3rd bar, the 'page header'. This bar needs a different approach as it has to become part of the layout Header and can't be part of the page itself. We'll need a different task and PR for that. I didn't exactly want to remove it yet as the styling and layout might get re-used, but since it's not being used right now and will need to be remade anyway I've opted to just delete the entire file.